plugins/source/datadog/resources/services/downtimes/downtimes.go

```diff
@@ -15,20 +14,9 @@ func Downtimes() *schema.Table {
 		Name:      "datadog_downtimes",
 		Resolver:  fetchDowntimes,
 		Multiplex: client.AccountMultiplex,
-		Transform: transformers.TransformWithStruct(&datadogV1.Downtime{}),
+		Transform: client.TransformWithStruct(&datadogV1.Downtime{}, transformers.WithPrimaryKeys("Id")),
```

Comment:
`transformers.WithPrimaryKeys("Id")` might be moved to `client.TransformWithStruct`

Reply:
I don't think that's wise, we'd need to revert it all for a single struct later on :)